taxonomy-api/taxonomy-actors/src/main/scala/org/sunbird/mangers/CategoryManager.scala

```diff
@@ -0,0 +1,26 @@
+package org.sunbird.mangers
+
+import org.sunbird.common.Platform
+import org.sunbird.common.dto.Request
+import org.sunbird.common.exception.ClientException
+
+import java.util
+import java.util.Optional
+import scala.collection.JavaConverters.mapAsScalaMapConverter
+import scala.collection.mutable.ListBuffer
+
+object CategoryManager {
+
+  def validateTranslationMap(request: Request) = {
+    val translations: util.Map[String, AnyRef] = Optional.ofNullable(request.get("translations").asInstanceOf[util.HashMap[String, AnyRef]]).orElse(new util.HashMap[String, AnyRef]())
+    if (translations.isEmpty) request.getRequest.remove("translations")
+    else {
+      val languageCodes = Platform.getStringList("platform.language.codes", new util.ArrayList[String]())
```

Comment:
Move this outside the method. It doesn't change. So, it should not be a method level variable.